roles/pgbench/tasks/run_workload.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Init workload start signal to false
+  command: "redis-cli set pgb_start false;"
+
+- name: Init client ready state to empty
+  command: "redis-cli del pgb_client_ready"
+
+- name: Setup pgbench test job(s)
+  k8s:
+    definition: "{{ lookup('template', 'templates/workload.yml.j2') }}"
+  with_indexed_items: "{{ pgbench.databases }}"
+  when: item.0|int in range(dbnum)
+
+- name: Wait for all clients to be ready
+  shell: "redis-cli --raw llen pgb_client_ready"
+  register: client_ready
+  #until: "client_ready.stdout|int == pgbench.databases|length"
```

Comment:
Can we nix this line? Doesn't seem needed.

Reply:
Whoops. Testing stuff that didn't get cleaned up.